[README.Android.md] Update Configure Commands

taoyouh opened the pull request:
## Description
<!--- Provide a general summary of your change in the Pull Request title above -->
<!--- Describe your change in detail here -->
Corrected the configure command in Android readme.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here -->
According to the steps above, the Android SDK binaries are in $HOME/android-tools/android-sdk-linux/tools/bin. Thus, the configure command should have its "--with-sdk-path" parameter set to that location.

## How Has This Been Tested?
<!--- Please describe in detail how you tested your change -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc -->
Using the original command, "configure" reports an error. The new command completes normally.

## Screenshots (if appropriate):
`--with-sdk-path=$HOME/android-tools/android-sdk-linux` produces:
```
ls: cannot access '/home/zhaoquan/android-tools/android-sdk-linux/build-tools': No such file or directory
checking for aapt... no
configure: error: "Missing program: aapt"
```

`--with-sdk-path=$HOME/android-tools/android-sdk-linux/tools/bin` succeeds

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

taoyouh committed:
Update Configure Commands in README.Android.md

According to the steps above, the Android SDK binaries are in $HOME/android-tools/android-sdk-linux/tools/bin. Thus, the configure command should have its "--with-sdk-path" parameter set to that location.

wsnipex commented:
this is wrong. If you follow the docs the correct path is e.g $HOME/android-sdk-linux/build-tools/28.0.3

taoyouh commented:
> this is wrong. If you follow the docs the correct path is e.g $HOME/android-sdk-linux/build-tools/28.0.3

It seems that I modified the steps a little. When I tried to follow the doc, I see this output from sdkmanager:
```
Warning: Could not create settings
java.lang.IllegalArgumentException
        at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.<init>(SdkManagerCliSettings.java:428)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.createSettings(SdkManagerCliSettings.java:152)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.createSettings(SdkManagerCliSettings.java:134)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:57)
        at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:48)
```
And then I added `--sdk-root=.` in the command, so my SDK root become `$HOME/android-tools/android-sdk-linux/tools/bin` and caused my problem.

According to this [stackoverflow question](https://stackoverflow.com/questions/60440509/android-command-line-tools-sdkmanager-always-shows-warning-could-not-create-se), the latest "command line tool" instead of "sdk tool" works differently and require additional steps than the doc said. Thus, the steps of configuring Android SDK need to be updated while this PR is wrong.

taoyouh commented:
I think the actual solution to the problem I met is #17834
